make/photon/db/upgrade.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+source $PWD/initdb.sh
+
+PGDATA="/var/lib/postgresql/data"
+PGBINOLD="/usr/local/pg96/bin/"
+PGBINNEW="/usr/bin"
+PGDATAOLD=""
+PGDATANEW=""
+
+while [[ "$#" -gt 0 ]]; do
+        case $1 in
+                -b|--old-datadir) PGDATAOLD="$2"; shift ;;
+                -B|--new-datadir) PGDATANEW="$2"; shift ;;
+                -d|--old-bindir) PGBINOLD="$2"; shift ;;
+                -D|--new-bindir) PGBINNEW="$2"; shift ;;
+                *) echo "Unknown parameter passed: $1"; exit 1 ;;
+        esac
+        shift
+done
+
+if [ "$PGDATAOLD" = "" ] || [ "$PGDATANEW" = "" ]; then
+        echo "Required parameter is missing: $PGDATAOLD, $PGDATANEW"
+        exit 1
+fi
+
+export PGDATAOLD=$PGDATAOLD
+export PGDATANEW=$PGDATANEW
+export PGBINNEW=$PGBINNEW
+export PGBINOLD=$PGBINOLD
+
+echo 'start to upgrade.'
+cd /tmp
```

Comment:
is this needed?

Reply:
the pg_upgrade log will be generated in the tmp folder, only failure will be printed.